apps/etterlatte-behandling/src/main/kotlin/Application.kt

```diff
@@ -74,10 +82,47 @@ private class Server(private val context: ApplicationContext) {
     fun run() =
         with(context) {
             dataSource.migrate()
+            populerSaksbendlereMedNavn(context)
             setReady().also { engine.start(true) }
         }
 }
 
+@OptIn(DelicateCoroutinesApi::class)
+private fun populerSaksbendlereMedNavn(context: ApplicationContext) {
+    if (context.leaderElectionKlient.isLeader()) {
+        GlobalScope.launch(newSingleThreadContext("saksbehandlernavnjob")) {
+            val logger = LoggerFactory.getLogger("saksbehandlernavnjob")
+            val tidbrukt =
+                measureTime {
+                    logger.info("Starter job for å legge inn saksbehandlere med navn")
+                    val sbidenter = context.saksbehandlerInfoDao.hentalleSaksbehandlere().mapNotNull { it }
```

Comment:
Litt uheldig å blande inn kobling mot `oppgave` tabellen i denne DAO-en.

Reply:
Ja, alternativt er en egen dao for oppgave kun til denne jobben da den vanlige daoen krever `Intransaction`